src/rideables/Stack.hpp

```diff
@@ -0,0 +1,98 @@
+// #ifndef STACK_HPP
+// #define STACK_HPP
```

Comment:
take these back, they help prevent the header file from being included more than once.